depends/README.md

```diff
@@ -30,8 +30,12 @@ Common `host-platform-triplets` for cross compilation are:
 - `aarch64-linux-gnu` for Linux ARM 64 bit
 - `riscv32-linux-gnu` for Linux RISC-V 32 bit
 - `riscv64-linux-gnu` for Linux RISC-V 64 bit
+- `aarch64-linux-android` for Android ARM 64 bit
 
-No other options are needed, the paths are automatically configured.
+The paths are automatically configured and no other options are needed unless targeting Android.
+In that case one needs to set `ANDROID_API_LEVEL` and `ANDROID_TOOLCHAIN_BIN`:
```

Comment:
Can you add a link where one can obtain the toolchain? I'm guess here: https://developer.android.com/ndk/downloads